components/cards/WhatWeDoCard.jsx

```diff
@@ -0,0 +1,38 @@
+import { motion, useAnimation } from "framer-motion";
+import { useInView } from "react-intersection-observer";
+import { useEffect } from "react";
+import Image from 'next/image'
+import { Heading4 } from '../ui/Headings';
+
+const WhatWeDoCard = ({ index, title, description, image }) => {
+    const boxVariant = {
+        visible: { opacity: 1, scale: 1, x: 0, transition: { duration: 0.5 } },
+        hidden: { opacity: 0, scale: 0, x: index % 2 == 0 ? 100 : -100 },
+    }
```

Comment:
This could be renamed `boxVariants` (in plural) as they are several variants of the box